.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+on: push
+jobs:
+  test:
+    name: Run tests
+    env:
+      COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tox: [flake8, safety, py37, coveralls]
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Are you sure we want to checkout master here? I'm still unclear on the configs so I may be missing things, but it feels like this would always checkout the master branch even on push to another branch?

Reply:
I know it looks like I'm checking out the master branch, but that's not what's happening. What that line does is use the master branch of the checkout action. It will checkout whatever version of the codebase triggered the workflow. There's a little more info here: https://github.com/actions/checkout#usage.